python/src/LinearModelAlgorithm_doc.i.in

```diff
@@ -5,7 +5,7 @@
 
     LinearModelAlgorithm(Xsample, Ysample)
 
-    LinearModelAlgorithm(Xsample, basis, Ysample)
+    LinearModelAlgorithm(Xsample, Ysample, basis)
```

Comment:
here you can drop the "Available usages" list since the 2 ctors only differ by an extra argument: just mention that the basis argument is optional in the parameters section

Reply:
ok